components/runners/ambient-runner/tests/test_workflow_endpoint.py

```
@@ -0,0 +1,200 @@
+        dest = args[-1] if args else kwargs.get("args", [None])[-1]
+        Path(dest).mkdir(parents=True, exist_ok=True)
+        return _make_successful_process()
+
+    with patch(
+        "asyncio.create_subprocess_exec", side_effect=fake_subprocess
+    ), patch(
+        "ambient_runner.endpoints.workflow.ensure_git_auth"
+    ), patch(
+        "os.getenv", side_effect=lambda k, d="": str(workspace) if k == "WORKSPACE_PATH" else d
+    ):
+        success, path = await clone_workflow_at_runtime(git_url, "main", "")
```

Ruff (0.15.17)

[warning] Unpacked variable `path` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)